inbox/api/validation.py

```diff
@@ -28,6 +28,23 @@ def bounded_str(value, key):
     return value
 
 
+def comma_separated_email_list(value, key):
+    addresses = value.split(',')
```

Comment:
Teeeechnically, `"foo,bar"@example.com` is a valid address... not sure if we need to consider that in practice